test/test_mysql_shared_helm_template.py

```diff
@@ -23,11 +23,11 @@ def teardown_method(self):
         self.hc_api.delete_project()
 
     def test_package_persistent(self):
-        self.hc_api.package_name = "mysql-imagestreams"
+        self.hc_api.package_name = "redhat-mysql-imagestreams"
         assert self.hc_api.helm_package()
         assert self.hc_api.helm_installation()
-        self.hc_api.package_name = "mysql-persistent"
+        self.hc_api.package_name = "redhat-mysql-persistent"
         assert self.hc_api.helm_package()
-        assert self.hc_api.helm_installation(values={".mysql_version": "8.0-el8", ".namespace": self.hc_api.namespace})
+        assert self.hc_api.helm_installation(values={"mysql_version": "8.0-el8", "namespace": self.hc_api.namespace})
```

Comment:
I'm not sure how this works, but is the `8.0-el8` correct here when you're enabling this for `8.4-el9`?

Reply:
Good point. I will update. Thanks.

Reply:
This is fixed by commit https://github.com/sclorg/mysql-container/pull/390/commits/6e8fbbdb127ae7d2baea4c3dcf1f6aeb90f98566